analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -17,19 +17,42 @@ library(dplyr)
 ### Read in data
 
 ```{r}
-cnvkit_file <- file.path("..", "..", "data", "cnv-cnvkit.seg.gz")
+cnvkit_file <- file.path("..", "..", "data", "v8", "cnv-cnvkit.seg.gz")
 cnvkit_df <- readr::read_tsv(cnvkit_file)
 ```
 
 ```{r}
-histologies_file <- file.path("..", "..", "data", "histologies.tsv")
+histologies_file <- file.path("..", "..", "data", "v8", "histologies.tsv")
 histologies_df <- readr::read_tsv(histologies_file, guess_max = 100000)
 ```
 
+
+# Edits for https://github.com/PediatricOpenTargets/ticket-tracker/issues/177
+# For PBTA, histologies file has germline_sex_estimate already assigned to WXS patients which have had WGS performed
+# Updating here germline_sex_estimate for cohort=="TARGET" where is.na(germline_sex_estimate)
+#Isolate IDs where germline_sex_estimate is NA and cohort == "TARGET"
+germline_sex_estimate_na <- histologies_df %>% filter(cohort == "TARGET") %>% filter(is.na(germline_sex_estimate))
```

Comment:
Actually as I was going through this more, I think here we need to do 
```
histologies_df %>% filter(experimental_strategy == "WXS") %>% 
filter(sample_type == "Tumor) %>%
filter(is.na(germline_sex_estimate))
```
I didn't add the cohort since we can do a cohort-less workflow per @jharenza's suggestion.

Reply:
I reviewed the code and implemented all the appropriate changes, and verified that it creates ```cnvkit_with_status.tsv``` in the ```scratch``` folder with expected values for ```germline_sex_estimate```.